.github/workflows/ci.yml

```diff
@@ -144,11 +141,17 @@ jobs:
           toolchain: stable
           override: true
 
-      - name: Run benches
+      - name: Run akd_core benches
         uses: actions-rs/cargo@v1
         with:
           command: bench
-          args: -F bench --no-run
+          args: -p akd_core -F bench --no-run
+
+      - name: Run akd benches
+        uses: actions-rs/cargo@v1
+        with:
+          command: bench
+          args: -p akd -F bench --no-run
```

Comment:
Can we use a [matrix strategy](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) here to parallelize this? Similar to the matrix strategy used for the main tests.